crates/ironrdp-client/src/app.rs

```diff
@@ -325,13 +348,10 @@ impl ApplicationHandler<RdpOutputEvent> for App {
             RdpOutputEvent::Image { buffer, width, height } => {
                 trace!(width = ?width, height = ?height, "Received image with size");
                 trace!(window_physical_size = ?window.inner_size(), "Drawing image to the window with size");
-                self.buffer_size = (width, height);
+                self.buffer_size = (width.get(), height.get());
                 self.buffer = buffer;
                 surface
-                    .resize(
-                        NonZeroU32::new(u32::from(width)).unwrap(),
-                        NonZeroU32::new(u32::from(height)).unwrap(),
-                    )
+                    .resize(NonZeroU32::from(width), NonZeroU32::from(height))
```

Comment:
I have changed the `RdpOutputEvent::Image`'s `width` and `height` types to `NonZero`, because I cannot directly propagate the error from this method. All I can do is print an error and exit from the event loop here. So, I made handling of the _Image_ event infallible.